releasenotes/notes/fix-e8168413eeec1e85.yaml

```diff
@@ -0,0 +1,6 @@
+---
+fixes:
```

Comment:
Calling this a "feature" instead of a "fix" could more accurately represent the change in addition to avoiding the need for backports.